android/src/main/java/com/swmansion/gesturehandler/core/NativeViewGestureHandler.kt

```diff
@@ -199,6 +201,11 @@ class NativeViewGestureHandler : GestureHandler<NativeViewGestureHandler>() {
      * by this one.
      */
     fun shouldCancelRootViewGestureHandlerIfNecessary() = false
+
+    /**
+     * Decides whether or not target view should pass event to children.
+     */
+    fun triggerEvent(view: View?, event: MotionEvent) = view?.onTouchEvent(event)
```

Comment:
```suggestion
    /**
     * Passes the event down to the underlying view using the correct method.
     */
    fun sendTouchEvent(view: View?, event: MotionEvent) = view?.onTouchEvent(event)
```

Reply:
Done in [4ca8726](https://github.com/software-mansion/react-native-gesture-handler/pull/3244/commits/4ca8726e9be7718b2c2211a17bfacb663d80e4a8)